### ISSUE TITLE
Maybe an onCollide event bug?

### ISSUE THREAD

**Issue by emmelleppi** [react-spring/use-cannon#65]
I think there is a bug in related to `onCollide` event.

The collision between two bodies works perfectly, the same can be said for the collisionFilters, but the `onCollide` event always returns as `body` and `target` the same object.
The same obviously applies to the `collisionFilters` of the event, `filtergroup` and `filtermask` of `body` and `target` are always the same.

In particular:
- `bodyA` collides with `bodyB`
- `onCollide` event of `bodyA` has` body = target = bodyA`
- `onCollide` event of `bodyB` has `body = target = bodyB`

**Comment by codynova** [comment]
This is indeed a bug. It looks like, during a "collide" event on a body, Cannon only returns that body which the event was attached to - but not the second body that was collided with.

I misunderstood this when updating the "collide" event behavior in this `use-cannon` lib, which is why there is both a "target" and "body" that are identical.

I can change the collide event to expose both collided bodies, but I'm wondering why it was done in this way to begin with...

**Comment by codynova** [comment]
This should now be resolved in 1f0cef6 and released in `use-cannon@0.4.2`

Note that `target` is the body with the event listener attached, and `body` is the *other* colliding body:

```js
sphereBody.addEventListener('collide', ({ target, body }) => {
  // where sphereBody === target
  console.log('Sphere collided with another body:', body)
})
```

**Comment by codynova** [comment]
This should be fixed so I'm going to close the issue, let me know if you see any problems
